terraform/main.tfvars

```diff
@@ -45,9 +45,10 @@ az = [
     "ap-southeast-2b"
 ]
 inst_type = "t2.micro"
-inst_key = ""
+inst_key = "awskey"
 ami_id = "ami-064db566f79006111"
 asg_desired_capac = "2"
 asg_max_size = "2"
 asg_min_size = "1"
-containerimage = "048355378787.dkr.ecr.ap-southeast-2.amazonaws.com/pr1-grp3-ecr:1a109f0"
\ No newline at end of file
+containerimage = "607961847144.dkr.ecr.ap-southeast-2.amazonaws.com/wordpress:e2ddee6"
```

Comment:
we can't create one ECR repo and pull the image from another.
on terraform/ecr/main.tf  line 1 you're creating a repo called "pr1-grp3-ecr" .. but here you're pull the image from another one (wordpress)
it will work on your machine, but won't work on anyone else's.

Reply:
yes, you are right

Reply:
I will fix this